hoot-core/src/main/cpp/hoot/core/algorithms/changeset/ChangesetReplacementCreator.cpp

```diff
@@ -515,6 +519,37 @@ void ChangesetReplacementCreator::_snapUnconnectedPostChangesetMapCropping(
   OsmMapWriterFactory::writeDebugMap(refMap, _changesetId + "-ref-connected-combined");
 }
 
+void ChangesetReplacementCreator::_cropMapForChangesetDerivation(
```

Comment:
Moved this down from the base class, as its not needed by the cut only implementation. The two don't really make much sense inheriting from each other now, so will get around to refactoring both out with a common abstract base class at some point.